*** frameworks/base/core/java/android/app/admin/DevicePolicyManager.java.orig	2019-10-21 23:53:41.000000000 -0300
--- frameworks/base/core/java/android/app/admin/DevicePolicyManager.java	2019-10-21 23:55:10.372083031 -0300
***************
*** 3367,3373 ****
              return 0;
          }
          // Kind-of arbitrary.
!         return 16;
      }
  
      /**
--- 3367,3373 ----
              return 0;
          }
          // Kind-of arbitrary.
!         return 64;
      }
  
      /**
