*** bootable/recovery/recovery_ui/device.cpp.orig	2019-10-21 23:46:58.292352000 -0300
--- bootable/recovery/recovery_ui/device.cpp	2019-10-21 23:47:32.620983607 -0300
***************
*** 26,36 ****
    { "Enter fastboot", Device::ENTER_FASTBOOT },
    { "Apply update from ADB", Device::APPLY_ADB_SIDELOAD },
    { "Apply update from SD card", Device::APPLY_SDCARD },
-   { "Wipe data/factory reset", Device::WIPE_DATA },
    { "Wipe cache partition", Device::WIPE_CACHE },
-   { "Mount /system", Device::MOUNT_SYSTEM },
-   { "View recovery logs", Device::VIEW_RECOVERY_LOGS },
-   { "Run graphics test", Device::RUN_GRAPHICS_TEST },
    { "Run locale test", Device::RUN_LOCALE_TEST },
    { "Enter rescue", Device::ENTER_RESCUE },
    { "Power off", Device::SHUTDOWN },
--- 26,32 ----
